### ISSUE TITLE
Request - Damage Reduction Percentage Configs

### ISSUE THREAD

**Issue by Boolyman** [KenyMylankca/EnhancedArmaments#18]
I am loving this mod, but the percentage damage reductions are WAY too overpowered. Getting something like +15% damage reduction on a piece of Leather basically makes it diamond quality. Just rolling decent rarities on a full leather armor set can make a player nearly invincible.  Please add a way for devs to adjust the damage reduction percentages on the armor, like you do for the weapon damage.  I know being invincible sounds like a good thing, but that doesnt fit the theme of every pack.

**Comment by KenyMylankca** [comment]
It's already nerfed. But,
did you even play the latest version of Enhanced Armaments and check is it still the same or not? if not, go do it.
If you already played the latest version and think damage reduction is too overpowered; did you play with rarity config file? Rarity damage configs also effect armor rarity on damage reduction. 
Assuming you did all and still not satisfied, i'm gonna nerf the armor damage reduction once again and last time.

**Comment by Boolyman** [comment]
Not only have I tested it, but I have watched multiple streamers playing my pack and receiving little to no damage because they spam crafted low tier armor to have high rarity.  I did make adjustments to the rarity configs for damage... as I was unaware that damage reduction was also included in this.  

Mods like Infernal Mobs, Scaling Health, Epic Siege Mobs, etc.. are very popular in packs like mine, MadPack 4, etc... So players need to be able to deal more damage... but that has nothing to do with them needing to become invincible as well.  In most games/packs the more damage a character is capable of, the LOWER their defensive stats.  But, of course this is your mod, I am just pleading for some consideration.
